hermes-common/src/main/java/pl/allegro/tech/hermes/common/time/ModifiableClock.java

```diff
@@ -0,0 +1,32 @@
+package pl.allegro.tech.hermes.common.time;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.temporal.ChronoUnit;
+
+import static java.time.Instant.now;
+
+public class ModifiableClock extends Clock {
```

Comment:
I guess this class should be in tests package

Reply:
Indeed. Fixed. Thanks for spotting that.